scripts/perf/vm_scripts/workflow_scripts/hello_world_script.sh

```diff
@@ -0,0 +1,8 @@
+#! bin/bash
+
+echo "Running hello_world_script"
+
+curl -L https://raw.githubusercontent.com/broadinstitute/cromwell/develop/centaur/src/main/resources/standardTestCases/hello/hello.wdl -o workflow.wdl
+curl -L https://raw.githubusercontent.com/broadinstitute/cromwell/develop/centaur/src/main/resources/standardTestCases/hello/hello.inputs -o workflow_inputs.json
+
+curl -X POST "http://localhost:8000/api/workflows/v1" -H "accept: application/json" -H "Content-Type: multipart/form-data" -F "workflowSource=@workflow.wdl" -F "workflowInputs=@workflow_inputs.json;type=application/json"
```

Comment:
TOL There's a lot of repetition in these scripts, but while N > 2 it seems too early in the evolution of the perf testing system to refactor for commonality just yet.